From 2a580656d01355c7fcd52d3441362bc0d3daf166 Mon Sep 17 00:00:00 2001
From: Gergely Nagy <algernon@madhouse-project.org>
Date: Tue, 23 Aug 2011 20:36:57 +0200
Subject: [PATCH] dh_installlogcheck: Add support for --name.

This patch makes dh_installlogcheck be similar to other helpers, like
dh_installlogrotate that already support a --name option: to install
the files as if they were installed by a different package.

Signed-off-by: Gergely Nagy <algernon@madhouse-project.org>
---
 dh_installlogcheck |   12 +++++++++++-
 1 files changed, 11 insertions(+), 1 deletions(-)

diff --git a/dh_installlogcheck b/dh_installlogcheck
index b6956fa..982e75f 100755
--- a/dh_installlogcheck
+++ b/dh_installlogcheck
@@ -39,6 +39,16 @@ subdirectories of F<etc/logcheck/> in package build directories.
 
 =back
 
+=head1 OPTIONS
+
+=over 4
+
+=item B<--name=>I<name>
+
+Look for files named F<debian/package.name.logcheck.*> and install
+them into the corresponding subdirectories of F<etc/logcheck/>, but
+use the specified name instead of that of the package.
+
 =cut
 
 init();
@@ -56,7 +66,7 @@ foreach my $package (@{$dh{DOPACKAGES}}) {
 			if (! -d "$tmp/etc/logcheck/$type") {
 				doit("install","-o",0,"-g",0,"-d","$tmp/etc/logcheck/$type");
 			}
-			my $packagenodot=$package; # run-parts..
+			my $packagenodot=pkgfilename($package); # run-parts..
 			$packagenodot=~s/\./_/g;
 			doit("install","-m","0644",$logcheck,"$tmp/etc/logcheck/$type/$packagenodot");
 		}
-- 
1.7.2.5

